spec/common-client-libs-sections.json

```
@@ -785,6 +785,13 @@
         "slug": "getchannels",
         "product": "realtime",
         "type": "function"
+      },
+      {
+        "id": "broadcast-message",
+        "title": "Broadcast a message",
+        "slug": "broadcastMessage",
```

Can we stick to all lowercase for consistency?
```suggestion
        "slug": "broadcastmessage",
```

Alternatively `kebab-case` (which is ideal), but it appears we don't use that everywhere, particularly not in realtime.